README.md

```diff
@@ -1,5 +1,18 @@
-# File Content Extraction Solution Pack
+# Release Information
 
-Provides a set of use case playbooks to work with the File Content Extraction Connector.
+* **Version**: 1.1.0
+* **Certified**: No
+* **Authored By**: Parag Khatavkar & Mahdi Naili (Fortinet SE)
+* **Compatible Version**: FortiSOAR v7.2.0 and above
 
-Documentation available [here](docs/README.md)
\ No newline at end of file
+# Overview
+
+Solution Pack examines Advisories sent by various organisations via email with attachments in PDF, Excel, CSV, and other formats. `File Metadata` and `File Content` is added to the record's Description. It notifies the count of IPs, Domain Name, Hashes, URLs, and so on if they are discovered via Email
+
+![](./docs/res/Description.png)
+
+
+# Next Steps
+
+| [Installation](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#installation) | [Configuration](https://github.com/solution-pack-file-content-extraction/blob/release/1.1.0/docs/setup.md#configuration) | [Usage](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/usage.md) | [Contents](https://github.com/fortinet-fortisoar/solution-pack-file-content-extraction/blob/release/1.1.0/docs/contents.md) |
```

Comment:
Doc URL of [Configuration] is wrong, `fortinet-fortisoar` is missing. @cs-ankit-gupta Please correct it

Reply:
Also, add release_notes.md
@cs-ankit-gupta

Reply:
Added release_notes.md